src/Core/Models/CustomCommands/CustomCommand.cs

```diff
@@ -19,13 +20,14 @@ public CustomCommand(string name)
             Features = new List<CustomCommandFeature>();
         }
 
-        public bool Matches(string name)
+        public bool Matches(string name, string[] args)
         {
             if (Features.Find(c => c is AliasFeature) is AliasFeature alias 
```

Comment:
We should probably cache the return value of `matchesArgs(...)` here, so we don't need to call it twice.